src/Microsoft.DotNet.ProjectModel/project.json

```diff
@@ -1,5 +1,5 @@
 {
-  "version": "1.0.0-rc4-*",
+  "version": "1.0.0-featmsbuild-*",
```

Comment:
ProjectModel was scheduled to RTM at first.  We thought that ASP.NET runtime needed it.  However, ProjectModel references NuGet, and NuGet wasn't RTM'ing.  So we talked to the ASP.NET team, and they said their tooling references ProjectModel.  The runtime only needs DependencyModel.

Since we had already labeled ProjectModel as "rc2" as part of the whole "rc2" release, we bumped its version to "rc3" for Preview2.  After Preview2, we bumped it again to "rc4".

So that's the story of ProjectModel.  It has been in a weird state, but has never RTM'd.  Bringing it in line with the rest of our tooling makes sense to me.